SECURITY.md

```diff
@@ -0,0 +1,37 @@
+
+# Security Policy
+
+The NEST-Simulator is a research oriented software and as such is expected to
+run mainly in well protected environments. In case it is found that this
+software can be used to violate security mechanisms, developers will try to
+provide patches that mitigate the risk of this misuse.
+
+The current policy is SECURITY.md in nest/nest-simulator master branch, which
+can also be found at <https://github.com/nest/nest-simulator/security/policy>.
```

Comment:
```suggestion
The policy can also be found at <https://github.com/nest/nest-simulator/security/policy>.
```

Should we also (automagically) mirror a rendered version of the SECURITY.md file to somewhere on the nest-simulator.org homepage?